src/app/views.py

```diff
@@ -117,7 +117,6 @@ def submitNewLicense(request):
                     licenseHeader = form.cleaned_data['licenseHeader']
                     licenseComments = form.cleaned_data['comments']
                     licenseText = form.cleaned_data['text']
-                    userEmail = form.cleaned_data['userEmail']
```

Comment:
You need to remove all other instances of this variable `userEmail` used in the below code.
Like line no 149
```
data['userEmail'] = userEmail
```
then line 163
```
licenseRequest = LicenseRequest(licenseAuthorName=licenseAuthorName, fullname=licenseName, shortIdentifier=licenseIdentifier,
                        submissionDatetime=now, userEmail=userEmail, notes=licenseNotes, xml=xml)
```

Reply:
@jlovejoy You can make `userEmail=""` (empty) or update the `LicenseRequest` model on 163 to pass userEmail as None/Empty. We can also change the underlying structure of `LicenseRequest` but let's not do it right now.

Reply:
see latest changes...